lib/services/abilities/abilities.ts

```diff
@@ -0,0 +1,218 @@
+import { AuthorType, ConversationType, Message, Staff }     from "@prisma/client"
```

Comment:
NOTE: this file was renamed and moved.

No new code was added here. You can ignore